awx/main/tests/functional/conftest.py

```diff
@@ -293,6 +329,18 @@ def org_credential(organization, credentialtype_aws):
                                      organization=organization)
 
 
+@pytest.fixture
+def external_credential(credentialtype_external):
+    return Credential.objects.create(credential_type=credentialtype_external, name='external-cred',
+                                     inputs={'url': 'http://testhost.com', 'token': 'secret1'})
+
+
+@pytest.fixture
+def other_external_credential(credentialtype_external):
+    return Credential.objects.create(credential_type=credentialtype_external, name='other-external-cred',
+                                     inputs={'url': 'http://testhost.com', 'token': 'secret2'})
```

Comment:
The fixture flow for these is very clear, and these are things we want to have ready to go in the fixture namespace 👍 

The only other thing I would prefer to add is a `credential_input_source` fixture (or whatever the most canonical name would be). A lot of your tests seem to be really long and linear, because you tend to create the input sources through the mock API first. Those would be more clear if they could just start out with the object taken from a generic fixture.